### PR TITLE
[HOLD] Cargo Borg draft  

### PR DESCRIPTION
**Pull request by Oyu-De**
## What Does This PR Do
**Writing before I lose this code, and just having it as a draft for discussion and, when the time comes, future development/additions**
Before we start, yes, cargo doesn't have much gameplay as we have discussed in discord about 3-4 months ago. Still, just wanted to make the draft for aformentioned reasons.

Introduces a new module selection option: Cargo Borg.
Cargo borg, *currently* have

- RCS (Rapid Cargo Sender)
- Package Wrapper
- Destination Tagger
- Hand Labeler
- Essential tools for borgos, like fire extinguisher and flash

Planned 
- "Arms" or some sort of module that allows cargo borgs to hold creates. Basically what Ripley for cargo was suppose to do.

## Why It's Good For The Game
Adds an alternative to *passive/chill* gameplay of Janiborg to Cyborg/Silicon, and allowing them to be part of cargo.
This can also insentivise cargo to use disposals distribution more due to borgo having rechargable package wrapper
Additionally "not yet implemented, still thinking" ability to hold crates in hands, allows for borgo to make deliveries directly, instead of relying on chute network, which might be broken, or unreliable M.U.L.E. 

## Testing
I booted a local server "no database" and tried the functionality. So far it seemed to work fine. Ofcourse any change will be tested.

## Changelog
:cl:
add: Cargo Borg module
add: Package Wrapper now has "energy storage" for borg use and refill pruposes
wip: Cargo Borg module
imageadd: Added Cargo Borg sprites " yet to be finished"
/:cl:

### CODE REVIEW

**Comment by Oyu-De** [comment]
heads up, I suck at spriting, if anyone wants to, feel free to send sprites for this. rn they just use normal grey ones

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -862,6 +875,11 @@
 	name = "Cable Synthesizer"
 	statpanel_name = "Cable"
 
+/datum/robot_energy_storage/packageWrap
+	name = "Package Wrapper Synthesizer"
+	statpanel_name = "Package Wrapper"
+
+
```
[the remark this answers is not in the document]
Comment (Oyu-De) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:882]: *actually scratch that Im just blind*

**Comment by smokincheez** [comment]
> heads up, I suck at spriting, if anyone wants to, feel free to send sprites for this. rn they just use normal grey ones

I could give it a shot. My discord is Smokie#8559 if you want to DM me about it.

**Comment by komihess13** [comment]
would be cool if it had a conveyer belt placer, wrench, multitool and crowbar. could help make a ton of setups. maybe also a way to store shipping manifests/papers in general? like a moving file cabinet.

**Comment by Oyu-De** [comment]
> would be cool if it had a conveyer belt placer, wrench, multitool and crowbar. could help make a ton of setups. maybe also a way to store shipping manifests/papers in general? like a moving file cabinet.

I thought about it, maybe a conveyer belt placer, but not sure on other tools, as they can be easily printed by crew, and it might turn cargo borg into a private but less effective engiborg
Will sit on this idea, especially with the conveyor belt, as that can be super usefull.

**Comment by Oyu-De** [comment]
this *should* fix the merge conflict. The branch was 4 months old.

**Comment by hal9000PR** [comment]
this thing better get a forklift

**Comment by Oyu-De** [comment]
> this thing better get a forklift

oh you know it will, tryna make it so it can carry crates, possibly 2 at a time.

**Comment by Denthamos** [comment]
> > this thing better get a forklift
> 
> oh you know it gonna be a glorified forklit

If it is able to hold a crate, the crate must be visible sprite wise.  Considering the speed borgs can move at with vtec upgrade, all access, and not being bulky/slow like a ripley which can also hold crates/lockers inside of it, being able to hold a crate that is effectively invisible would be a bit too strong with none of the downsides of the ripley.

**Comment by McRamon** [comment]
> If it is able to hold a crate, the crate must be visible sprite wise. Considering the speed borgs can move at with vtec upgrade, all access, and not being bulky/slow like a ripley which can also hold crates/lockers inside of it, being able to hold a crate that is effectively invisible would be a bit too strong with none of the downsides of the ripley.
> 
> _Also why do we not have an actual forklift for cargo yet to move crates around_

Well, while pulling a crate makes everything else kinda useless, unless we give forklift tool/machine an ability to lift multiple crates, there is not mcuh point in the forklift. And donwside of a borg in that case i guess would be that you have to emag it first before you do al the sneaky crate moving. Non-antag crate moving being done by disposals anyway.
Though visible crate is good just for the sake of visualisation.

**Comment by hal9000PR** [comment]
Can I get an update on the status of this?

**Pull request closed without merging**